portfolio/src/main/webapp/math.js

```diff
@@ -17,6 +18,7 @@ window.onload = function setUpGame() {
 
 function startGame() {
     question.textContent = generateQ();
+    //Remove intro screen and show questions when game starts
```

Comment:
In general, we want comments to be well formatted English. NIT, but have introduce a space between the comment and the R in Remove, and also missing a period.

// Remove intro screen and show questions when game starts.

Reply:
Thanks for the info! Fixed.